### PR TITLE
LC-360 Connector: BlankConnector

### PR DESCRIPTION
**Pull request by abijitran**
Created a connector that generates blank documents when given specified doc amount

### CODE REVIEW

File: lucille-core/src/main/java/com/kmwllc/lucille/connector/BlankConnector.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Connector implementation that produces blank documents given amount to produce
+ */
+public class BlankConnector extends AbstractConnector {
+
+  private static final Logger log = LoggerFactory.getLogger(BlankConnector.class);
+  private final int docNum;
```
Comment (rseitz) [inline lucille-core/src/main/java/com/kmwllc/lucille/connector/BlankConnector.java:16]: suggest renaming to numDocs

File: lucille-core/src/main/java/com/kmwllc/lucille/connector/BlankConnector.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Publisher;
+import com.typesafe.config.Config;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Connector implementation that produces blank documents given amount to produce
+ */
+public class BlankConnector extends AbstractConnector {
```
Comment (rseitz) [inline lucille-core/src/main/java/com/kmwllc/lucille/connector/BlankConnector.java:13]: decision was to rename to SequenceConnector

File: lucille-core/src/main/java/com/kmwllc/lucille/connector/BlankConnector.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  private static final Logger log = LoggerFactory.getLogger(BlankConnector.class);
+  private final int docNum;
+  private final int docStartNum;
```
Comment (rseitz) [inline lucille-core/src/main/java/com/kmwllc/lucille/connector/BlankConnector.java:17]: could rename to startWith or sequenceMinValue

Comment (abijitran) [reply lucille-core/src/main/java/com/kmwllc/lucille/connector/BlankConnector.java]: done

File: lucille-core/src/main/java/com/kmwllc/lucille/connector/BlankConnector.java (hunk 2 above)
Comment (abijitran) [reply lucille-core/src/main/java/com/kmwllc/lucille/connector/BlankConnector.java]: done

File: lucille-core/src/main/java/com/kmwllc/lucille/connector/BlankConnector.java (hunk 1 above)
Comment (abijitran) [reply lucille-core/src/main/java/com/kmwllc/lucille/connector/BlankConnector.java]: done

**Review by rseitz — APPROVED** [review]

**Pull request merged**